certified-connectors/Tesseron Invoice/apiDefinition.swagger.json

```diff
@@ -0,0 +1,245 @@
+            }
+          }
+        ]
+      }
+    },
+    "/CreateActivityRecording": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Message": {
+                  "type": "string",
+                  "description": "Message"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "ActivityRecordingId": {
+                  "type": "string",
+                  "description": "Activity Recording ID"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create Activity Recording",
+        "description": "This method allows you to create activity recordings.",
+        "operationId": "CreateActivityRecording",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "dateFrom": {
+                  "type": "string",
+                  "description": "Specify the start date",
+                  "title": "Date from"
+                },
+                "dateTo": {
+                  "type": "string",
+                  "description": "Specify the end date",
+                  "title": "Date to"
+                },
+                "bookText": {
+                  "type": "string",
+                  "description": "Specify a booking text for your activity recording",
+                  "title": "Booking Text"
+                },
+                "noteText": {
+                  "type": "string",
+                  "description": "Specify a note text for your activity recording",
+                  "title": "Note Text"
+                },
+                "ticketID": {
+                  "type": "integer",
+                  "description": "Specify a ticket id",
+                  "title": "Ticket ID"
+                },
+                "projectId": {
+                  "type": "integer",
+                  "description": "Specify a project id",
+                  "title": "Project ID"
+                },
+                "projectPhaseId": {
+                  "type": "integer",
+                  "description": "Specify a project phase id",
+                  "title": "Project Phase ID"
+                },
+                "projectPhaseTaskId": {
+                  "type": "integer",
+                  "description": "Specify a project phase task id",
+                  "title": "Project Phase Task ID"
+                }
+              },
+              "required": [
+                "dateFrom",
+                "dateTo",
+                "noInvoice",
+                "extraCharge"
```

Comment:
@developmentLL The required property 'extraCharge' is not defined in the object schema. May I ask you please to define it?